index.html

```diff
@@ -1,277 +1,334 @@
-      <div class="card">
-        <h3>Profesional Art Printing Data</h3>
-        <p class="description">A daily selection of privately personalized reads; no accounts or sign-ups required. has been the industry's standard</p>
-        <ul class="card-ul">
-          <li>html</li>
-          <li>bootstrap</li>
-          <li>ruby</li>
-        </ul>
-        <button type="button" class="card-button">See Project</button>
-      </div>
-    </section>
-    <section class="work-cards">
-      <div class="card">
-        <h3>Profesional Art Printing Data</h3>
-        <p class="description">A daily selection of privately personalized reads; no accounts or sign-ups required. has been the industry's standard</p>
-        <ul class="card-ul">
-          <li>html</li>
-          <li>bootstrap</li>
-          <li>ruby</li>
-        </ul>
-        <button type="button" class="card-button">See Project</button>
-      </div>
-    </section>
-    <section class="work-cards">
-      <div class="card">
-        <h3>Profesional Art Printing Data</h3>
-        <p class="description">A daily selection of privately personalized reads; no accounts or sign-ups required. has been the industry's standard</p>
-        <ul class="card-ul">
-          <li>html</li>
-          <li>bootstrap</li>
-          <li>ruby</li>
-        </ul>
-        <button type="button" class="card-button">See Project</button>
-      </div>
-    </section> 
-    <section class="work-cards">
-      <div class="card">
-        <h3>Profesional Art Printing Data</h3>
-        <p class="description">A daily selection of privately personalized reads; no accounts or sign-ups required. has been the industry's standard</p>
-        <ul class="card-ul">
-          <li>html</li>
-          <li>bootstrap</li>
-          <li>ruby</li>
-        </ul>
-        <button type="button" class="card-button">See Project</button>
-      </div>
-    </section> 
+    <!-- headline-container end -->
+
+
+      <!-- My Recent Works section start  -->
+      <section class="work">
+        <div class="work-header">
+          
+          <h2 class="work-title">My Recent Works</h2>
+          <div class="work-header-img"></div>
+          <img src="images/work-title-img.svg" alt="">
+        </div>
+        <div class="work-body">
+         <img src="#" alt="">
+          <article class="work-article">
+             <h2>Multi-Post Stories</h2>
+             <p>A daily selection of privately personalized reads; no accounts or sign-ups required. has text been the industry's standard dummy ever since the 1500s, when an unknown printer took a standard dummy text.</p>
+             <ul class="work-links">
+              <li><a href="#">Html</a></li>
+              <li><a href="#">Css</a></li>
+              <li><a href="#">Javascript</a></li>
+              <li><a href="#">Ruby</a></li>
+             
+             </ul>
+             <button type="button" class="work-see-button">See Project</button>
```

Comment:
OK @SAZZAD1Q2, So I think you misunderstood this design implementation. Let me clarify you a little bit about this Design element.
So there would be no `See Project` button on the work card, until you hover over it.
![image](https://user-images.githubusercontent.com/61361037/221350399-39f40d53-840e-4a4a-8a64-be3163f0dab3.png)
But when you hover any of the elements, then this button would appear.
![image](https://user-images.githubusercontent.com/61361037/221350449-3d625a90-547c-450f-87fb-572005fc4cf4.png)